src/main/java/cloud/fogbow/accs/core/models/orders/ComputeOrder.java

```diff
@@ -6,6 +6,8 @@
 import javax.validation.constraints.Size;
 import java.util.*;
 
+// TODO explain why this class is duplicated in RAS and ACCS or remove
+// this version
```

Comment:
Deal with the ToDos.

Reply:
Duplicated RAS models are removed by PR https://github.com/fogbow/accounting-service/pull/32.

Reply:
Done